templates/settings-form-application.hbs

```diff
@@ -186,15 +186,15 @@
 
         {{#if show_advanced}}
             <div class="form-group">
-                <h3>
+                <h4>
                     <label for="combat_tracking_enabled"
                            title="(Under Development) Oronder will post rolls from Discord to Foundry if a GM is logged into Foundry.">
```

Comment:
I changed this to `Enable Combat Tracking`. It was set up programmatically as enabling the combat tracking.
Could we separate the `Publish Discord Rolls` into another checkbox for combat? Since someone might want to enable combat tracking but not have the rolls come from discord.